### PR TITLE
added basic test for getFinalCsvRow

### PR DESCRIPTION
**Pull request by ryancwalsh**

### CODE REVIEW

File: db/Row.test.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// Run via `yarn test db/Row.test.ts`
+
+import { getFinalCsvRow } from './Row';
+
+// eslint-disable-next-line max-lines-per-function
+describe('Row', () => {
+  // eslint-disable-next-line max-lines-per-function
+  test('getFinalCsvRow', () => {
+    const accountId = 'a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48.factory.bridge.near';
+    const nearAmount = 33;
```
Comment (PierreLeGuen) [inline db/Row.test.ts:10]: Would be good to add a test for a negative NEAR value.

**Review by PierreLeGuen — CHANGES_REQUESTED** [review]

File: db/Row.test.ts (hunk 1 above)
Comment (ryancwalsh) [reply db/Row.test.ts:10]: Good idea! @PierreLeGuen 

I think this would be a case where (according to https://google.github.io/eng-practices/review/reviewer/) this should be an "FYI" for a future PR rather than a blocker here, right?

So ordinarily I'd want to go ahead and merge this.

However, in this case, when I opened this file again, my eslint flagged something that I wonder why it didn't flag before: I think `block_timestamp: number;` is an incorrect type. 

I think a lot of our types are incorrect.

So I created https://near-foundation.atlassian.net/browse/TTA-63

Maybe merging this now makes sense anyway since (like the Google guidelines say) this is progress. Thoughts?

cc @nearken

Comment (PierreLeGuen) [reply db/Row.test.ts:10]: @ryancwalsh  https://google.github.io/eng-practices/review/reviewer/looking-for.html#tests

I feel like this test is missing a primordial test case for it to be valid. It should fail if the NEAR amount transferred is returned positive instead of negative.

Comment (ryancwalsh) [reply db/Row.test.ts:10]: @PierreLeGuen I just read that tests link, but I don't know whether I'm following you. 

Please let me know if I'm misunderstanding something. 😃

a) if I found a different transaction case where the amount was negative and added a new test for that, it actually wouldn't change the flow of the implementation since `nearAmount`, `ftAmountIn`, and `ftAmountOut` all simply pass through (except in the case of staking). So adding that other case wouldn't be testing more routes of how the code can flow.

b) Your general point may be "This test is testing just 1 tiny case, and there are many, many cases that it isn't testing". Definitely. We can add those later.  It's not at all comprehensive yet.

So far, our team hasn't added many tests because we've been trying to quickly deliver at least _something_ that FLO can use.

I'm trying to at least get the framework in place so that we can start adding tests more easily and get in that habit.

**Review by PierreLeGuen — APPROVED** [review]
LGTM. We will add more tests later.
